dnsmasq-plan/README.md

```diff
@@ -0,0 +1,29 @@
+# dnsmasq
+
+This packages provides [dnsmasq](http://www.thekelleys.org.uk/dnsmasq/doc.html).
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Type of Package
+
+Binary package
```

Comment:
As you're including a `pkg_svc_run` line, this makes the package a `Service package`, and should then include additional documentation around Update strategy, Topology etc, reference [here](https://github.com/habitat-sh/core-plans/blob/master/README_TEMPLATE_FOR_PLANS.md).

Additionally, the plan should export/expose ports as required so that it can be service bound, or exposed in a container.